app/components/manageable_authenticator_component.html.erb

```diff
@@ -0,0 +1,66 @@
+<%= content_tag(
+      :'lg-manageable-authenticator',
+      **tag_options,
+      role: 'list-item',
+      'api-url': manage_api_url,
+    ) do %>
+  <div class="manageable-authenticator__edit" tabindex="-1">
+    <%= render AlertComponent.new(type: :error, class: 'manageable-authenticator__error-alert') %>
+    <form class="manageable-authenticator__rename">
+      <input
+        class="manageable-authenticator__rename-input usa-input width-full"
+        aria-label="<%= t('components.manageable_authenticator.nickname') %>"
+        value="<%= configuration.name %>">
```

Comment:
I'll change it to use a tag builder, but you've piqued my interest: What would be an example attack here? Even if our CSP allowed unsafe inline scripts (it doesn't), Rails seems to escape any quotes or HTML tag delimiters, and the attribute value itself isn't prone to evaluating any JavaScript that I'm aware of?

Reply:
Oh right I forgot that strings are HTML-unsafe by default. But one thing about Rails HTML sanitizarion is that it doesn't understand context, because escaping HTML entities for the body of a tag is slightly different than escaping for HTML attributes. I'll see if I can look up some ways to break through, but I don't remember off the top of my head